overlord/snapstate/handlers_test.go

```diff
@@ -114,3 +114,105 @@ func (s *handlersSuite) TestSetTaskSnapSetupLaterTask(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(newsnapsup.Channel, Equals, snapsup.Channel)
 }
+
+func (s *handlersSuite) TestComputeMissingDisabledServices(c *C) {
+	for _, tt := range []struct {
+		stDisabledSvcsList []string
+		missing            []string
+		found              []string
+		err                error
+		apps               map[string]*snap.AppInfo
```

Comment:
Nitpick, I'd move `apps` next to `stDisabledSvcsList`, so that all inputs are before expected outputs (and reflect agrs of `MissingDisabledServices(...)` call).

Reply:
Done